index.js

```diff
@@ -1,17 +1,49 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.length;
```

Comment:
```suggestion
    this.length = this.items.length;
```

creo que no lo viste pero si le pones el length ahi, siempre estara metiendo el mismo length una y otra vez, y eso no es verdad porque siempre tendria 0